realwork/Podfile.lock

```diff
@@ -2,20 +2,15 @@ PODS:
   - NLibrary (0.1.0)
 
 DEPENDENCIES:
-  - NLibrary (from `https://github.com/NathanDVT/NLibrary.git`)
+  - NLibrary (from `/Users/nngobale/Documents/Training 2020/Pod_Folder/NLibrary/NLibrary`)
 
 EXTERNAL SOURCES:
   NLibrary:
-    :git: https://github.com/NathanDVT/NLibrary.git
-
-CHECKOUT OPTIONS:
-  NLibrary:
-    :commit: 149836493a96bc7777d9b3c4873f942e25b8d921
-    :git: https://github.com/NathanDVT/NLibrary.git
+    :path: "/Users/nngobale/Documents/Training 2020/Pod_Folder/NLibrary/NLibrary"
 
 SPEC CHECKSUMS:
   NLibrary: ec2ddce5738362f9b660f27b78723df7e5fe7766
 
-PODFILE CHECKSUM: 4770deebebd51abf54033475bffbabadc656aa4f
+PODFILE CHECKSUM: a39daa594c047f1ff898c3094e7d90160b7441d8
```

Comment:
All of this is effected by the change to the pod file. Run pod update with the pod file pointing to a git repo